### ISSUE TITLE
Unable to tab to edit/delete icons in Names element of reports

### ISSUE THREAD

**Issue by csmith-he** [archesproject/Arches_HER#1038]
Example used is Consultations report

**Steps to Reproduce:**
-Login Navigate to an active Consultation 
-On the Consultation details page using the tab key navigate to the icons under Actions

**Expected Result:**
User should be able to tab to the icons under Actions

**Actual Result:**
User cannot tab to the icons under the Actions
